doc_admin/AdminManual.adoc

```diff
@@ -614,6 +618,11 @@ image:OH-main_menu_logo.png[OH main menu]
 
 The custom logo has to be a 100x100 pixel png image file.
 
+=== 2.2 OH - SERVER mode (portable)
+
+Starting with version 1.12, Open Hospital can be lauched in SERVER mode. SERVER mode allows to launch a local instance of the database server and listen for connections from other clients.
```

Comment:
I know I'm a broken record but didn't we agree not to use "Starting with version xxx" in the text as the document always matches the version of the code?

Reply:
Yes,right, also reworked a little bit, thank you !